kouncil-backend/src/main/java/com/consdata/kouncil/clusters/ClusterController.java

```diff
@@ -46,4 +47,10 @@ public boolean testConnection(@RequestBody ClusterDto cluster) {
     public boolean isClusterNameUnique(@PathVariable("clusterName") String clusterName) {
         return clusterService.isClusterNameUnique(clusterName);
     }
+
+    @RolesAllowed(Fields.CLUSTER_DELETE)
+    @DeleteMapping(path = "/api/cluster/{id}")
```

Comment:
Maybe it's good idea to add "/api/cluster" path as the @RequestMapping() to avoid code duplication?